conf/deployment/vsphere/upi_1az_rhcos_vsan_lso_vmdk_3m_6w_arbiter.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Would it be possible to have a more generic config file, for arbiter, which does not have any platform related params?
This way, we won't need an arbiter config file for each platform

Reply:
Arbiter deployment depends a lot on properties of openshift deployment (zones and their names, number of machines, LSO). So other deployment options needs to be aligned with arbiter options in ocs-ci config file, and for this reason it's not possible to just have a simple config file which would enable arbiter mode on top of any deployment.

Reply:
ok, thanks for the eplanation